tests/vppoutputencode.cpp

```diff
@@ -98,6 +103,17 @@ static void setEncodeParam(const SharedPtr<IVideoEncoder>& encoder,
     encoder->getParameters(VideoParamsTypeAVC, &encVideoParamsAVC);
     encVideoParamsAVC.idrInterval = encParam->idrInterval;
     encVideoParamsAVC.size = sizeof(VideoParamsAVC);
+#if YAMI_CHECK_API_VERSION(0, 2, 1)
```

Comment:
It is a tradeoff, check it in configure will introduce huge efforts to bisect.
Suppose I find an issue, I need check which commit bring this in.  Use this approach we do not need change yami-utils. We can just bisect yami core library. However, if we use configure approach. We need find every workable pair of yami core library and yami-utils. It is a nightmare for bug tracker.